app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenController.kt

```diff
@@ -147,10 +146,11 @@ class PatientSummaryScreenController @Inject constructor(
                 val timestamp = timestampGenerator.generate(measurement.createdAt)
                 SummaryBloodPressureListItem(
                     measurement = measurement,
-                    timestamp = timestamp,
+                    displayDaysTimestamp = timestamp,
                     isEditable = isBpEditable(measurement, bpEditableFor),
                     showDivider = measurement.uuid == lastElement.uuid,
-                    displayTime = displayTime(measurement.createdAt)
+                    displayTime = if (measurementList.size > 1) displayTime(measurement.createdAt) else null,
+                    firstInGroup = measurement.uuid == measurementList.first().uuid
                 )
```

Comment:
Doing an object comparison here is needless here. We just need to identify the last or first item in the list. Since it will happen for each element in the list, I preferred using `uuid`.

Reply:
It's less code to read. Don't worry about doing micro optimizations like these.